pose_detector.py

```diff
@@ -12,13 +7,20 @@
 import os
 from math import sin, cos
 from operator import add
+# from depth_data_generator import DepthDataGenerator
+# from depth_live_generator import DepthLiveGenerator
+from image_generator import ImageGenerator
+from video_file_generator import VideoFileGenerator
+from video_live_generator import VideoLiveGenerator
+from loading_bay_detector import LoadingBayDetector
+from power_port_detector import PowerPortDetector
 
 
 # finds rotation and translation of vision targets
 class PoseDetector:
```

Comment:
I just realized that naming the the situational awareness code *Detector is a bit confusing. Not a huge issue though.

Reply:
I wasn't sure what to name it, would PoseCalculator be better?